suite-common/trading/src/thunks/buy/confirmTradeThunk.ts

```diff
@@ -16,13 +15,20 @@ export type ConfirmTradeThunkProps = {
     address: string;
     account: Account;
 
+    analyticsTriggerConfirm: () => void;
     processResponseData: (response: BuyTradeResponse) => void;
 };
 
 export const confirmTradeThunk = createThunk(
     `${BUY_THUNK_COMMON_PREFIX}/confirmTrade`,
     async (
-        { returnUrl, address, account, processResponseData }: ConfirmTradeThunkProps,
+        {
+            returnUrl,
+            address,
+            account,
+            analyticsTriggerConfirm,
```

Comment:
nit: I would use something starting with verb

```suggestion
            triggerAnalyticsTradeConfirmation,
```

Reply:
fixed